pkg/webhook/admission/response.go

```diff
@@ -60,21 +60,24 @@ func Errored(code int32, err error) Response {
 }
 
 // ValidationResponse returns a response for admitting a request.
-func ValidationResponse(allowed bool, reason string) Response {
+func ValidationResponse(allowed bool, message string) Response {
 	code := http.StatusForbidden
+	reason := metav1.StatusReasonForbidden
 	if allowed {
 		code = http.StatusOK
+		reason = ""
 	}
 	resp := Response{
 		AdmissionResponse: admissionv1.AdmissionResponse{
 			Allowed: allowed,
 			Result: &metav1.Status{
-				Code: int32(code),
+				Code:   int32(code),
+				Reason: reason,
 			},
 		},
 	}
-	if len(reason) > 0 {
-		resp.Result.Reason = metav1.StatusReason(reason)
+	if len(message) > 0 {
```

Comment:
Nit: Do we need to check the length of message or can we just always set `resp.Result.Message`? `metav1.Status.Message` is a string type, so leaving it "unset" is the same as setting it to an empty string.

Reply:
You're right, we no longer need to check the length here. Is it worth changing or should we keep it as-is if there aren't other changes to be made?